enterprise_access/apps/api/serializers.py

```diff
@@ -141,7 +148,69 @@ def validate_content_key(self, value):
 
         return value
 
-class SubsidiyAccessPolicyListSerializer(SubsidiyAccessPolicyRedeemSerializer):  # pylint: disable=abstract-method
+
+class SubsidyAccessPolicyCRUDSerializer(serializers.ModelSerializer):
+    """
+    Serializer to validate policy data for CRUD operations.
+    """
+    uuid = serializers.UUIDField(read_only=True)
+    policy_type = serializers.ChoiceField(choices=PolicyTypes.CHOICES)
```

Comment:
`policy_type` is kind of private field and used to track the type of the actual(`proxy`) class of the policy. This should not be publicly exposed.

Reply:
I think we would need this to create an object of specific proxy type. Lets have a call tomorrow morning to discuss this.

Reply:
Yep, we would also need this to add validation logic around "showing relevant limit field based on selected policy type "